src/main/resources/assets/js/components/AboutPageComponent.jsx

```diff
@@ -0,0 +1,70 @@
+/** @jsx React.DOM */
+
+var React = require("react/addons");
+var Info = require("../models/Info");
+var BackboneMixin = require("../mixins/BackboneMixin");
+var PageComponent = require("../components/PageComponent");
+var ObjectDlComponent = require("../components/ObjectDlComponent");
+
+/* jshint trailing:false, quotmark:false, newcap:false */
+/* jscs:disable disallowTrailingWhitespace, validateQuoteMarks, maximumLineLength */
+var UNSPECIFIED_NODE = React.createClass({
+  render: function () {
+    return <span className="text-muted">Unspecified</span>;
+  }
+});
+
+var AboutPageComponent = React.createClass({
+  mixins: [BackboneMixin],
+
+  getInitialState: function () {
+    return {
+      info: new Info()
+    };
+  },
+
+  componentDidMount: function () {
+    this.state.info.fetch();
+  },
+
+  getResource: function () {
+    return this.state.info;
+  },
+
+  render: function () {
+    var marathonConfig = this.state.info.get("marathon_config");
+    var zookeeperConfig = this.state.info.get("zookeeper_config");
+
+    /* jshint trailing:false, quotmark:false, newcap:false */
+    /* jscs:disable disallowTrailingWhitespace, validateQuoteMarks, maximumLineLength */
+    return (
+      <PageComponent>
+        <h2>
+          <small className="text-muted">
+            Version {this.state.info.get("version")}
+          </small>
+        </h2>
+        <dl className="dl-horizontal dl-horizontal-lg">
+          <dt title="framework_id">Framework Id</dt>
+          <dd>
+            {this.state.info.get("framework_id") || <UNSPECIFIED_NODE />}
```

Comment:
I'd like to see this refactored into a function. Something like:

    function(attr) { return this.state.info.get(attr) || <UNSPECIFIED_NODE /> }